cli/msg.fbs

```diff
@@ -136,6 +142,13 @@ enum ErrorKind: byte {
   OpNotAvaiable,
   WorkerInitFailed,
   UnixError,
+
+  // binding errors
+  BindingError,
```

Comment:
Is this one used? Suggest removing.

Reply:
Yes errors from binding functions are `BindingError`s.